### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by sturkmen72**
### This pullrequest changes

minor changes

### CODE REVIEW

File: modules/tracking/src/trackerBoosting.cpp [hunk 1]
```diff
@@ -126,7 +126,7 @@ bool TrackerBoostingImpl::initImpl( const Mat& image, const Rect2d& boundingBox
   Mat_<int> intImage;
   Mat_<double> intSqImage;
   Mat image_;
-  cvtColor( image, image_, CV_RGB2GRAY );
+  cvtColor( image, image_, COLOR_BGR2GRAY );
```
Comment (sturkmen72) [inline modules/tracking/src/trackerBoosting.cpp]: (RFC) should i keep it `COLOR_RGB2GRAY`

Comment (alalek) [reply modules/tracking/src/trackerBoosting.cpp]: Yes, lets avoid using of legacy "_c.h" headers and their symbols.

Comment (sturkmen72) [reply modules/tracking/src/trackerBoosting.cpp]: R-G-B & B-G-R

Comment (alalek) [reply modules/tracking/src/trackerBoosting.cpp]: Oops...
Looks good to me.
OpenCV default format for I/O is `BGR`
(except Android - RGBA - they should explicitly pass gray images for such algorithms).

File: modules/xphoto/samples/inpainting.cpp [hunk 2]
```diff
@@ -41,22 +38,22 @@ int main( int argc, const char** argv )
...
-    cv::Mat src = cv::imread(inFilename, -1);
+    cv::Mat src = cv::imread(inFilename, cv::IMREAD_UNCHANGED);
     if ( src.empty() )
     {
         printf( "Cannot read image file: %s\n", inFilename.c_str() );
         return -1;
     }
 
-    cv::cvtColor(src, src, CV_RGB2Lab);
+    cv::cvtColor(src, src, cv::COLOR_RGB2Lab);
```
Comment (alalek) [inline modules/xphoto/samples/inpainting.cpp]: BGR too here and in `COLOR_Lab2RGB`

**Pull request merged**